rpc/coretypes/responses.go

```diff
@@ -241,7 +241,9 @@ type ResultConsensusState struct {
 type ResultBroadcastTx struct {
 	Code      uint32         `json:"code"`
 	Data      bytes.HexBytes `json:"data"`
+	Log       string         `json:"log"`
```

Comment:
is this needed?

Reply:
@LCyson do you remember whether this was added for debugging or needed for ibc to work?

Reply:
Yes this is needed, otherwise hermes will break bc it couldn't find the log

Reply:
hm it seems that we removed this^ we prob want to add it back